components/status_dropdown/status_dropdown.jsx

```diff
@@ -135,6 +135,7 @@ export default class StatusDropdown extends React.Component {
                 <Menu
                     ariaLabel={localizeMessage('status_dropdown.menuAriaLabel', 'Set a status')}
                     id='statusDropdownMenu'
+                    customStyles={{marginLeft: '6px'}}
```

Comment:
Can we keep the styles in the css files?

Reply:
Done.